tests/unit/test_cost.py

```diff
@@ -0,0 +1,5 @@
+from gh_actions_exporter.cost import Cost
+
+
+def test_get_job_cost(workflow_job, settings):
+    assert Cost(settings).get_job_cost(workflow_job, "large") == 0.16
```

Comment:
Much better.

Try adding a test case with a label that is not specified inside settings.job_costs